src/Backups/BackupsWorker.cpp

```diff
@@ -44,7 +45,7 @@ namespace ErrorCodes
     extern const int CONCURRENT_ACCESS_NOT_SUPPORTED;
 }
 
-using OperationID = BackupsWorker::OperationID;
+using OperationID = BackupOperationInfo::ID;
```

Comment:
Maybe it's better to declare `BackupOperationID` instead of `BackupOperationInfo::ID` just to be shorter.